pkg/security/tests/activity_dumps_test.go

```diff
@@ -31,6 +31,9 @@ func TestActivityDumps(t *testing.T) {
 	if testEnvironment == DockerEnvironment {
 		t.Skip("Skip test spawning docker containers on docker")
 	}
+	checkKernelCompatibility(t, "RHEL dont't have docker installed", func(kv *kernel.Version) bool {
```

Comment:
I think you can just use `whichNonFatal("docker")` and test the returned error to skip the test rather than testing each kernel individually.

Reply:
Updated on last commit! Thank you, it's clearer now